src/clientManager.js

```diff
@@ -3,13 +3,31 @@ var fs = require("fs");
 var Client = require("./client");
 var mkdirp = require("mkdirp");
 var Helper = require("./helper");
+var path = require("path");
+
+var USERS_PATH = path.join(Helper.HOME, "users");
```

Comment:
Heh, I went that way as your two new functions didn't use `prototype`, but others in the file do.